docs/guide/content_rendering.md

```diff
@@ -979,11 +979,11 @@ Typical use cases include access to:
 - Main Location and alternative Locations for the current Content item
 - etc.
 
-There are three ways in which you can apply a custom controller:
+There are three ways in which you can apply a custom logic:
 
-- Configure a custom controller alongside regular matcher rules to use **both** your custom controller and the `ViewController` (recommended).
-- [**Override**](#overriding-the-built-in-viewcontroller) the built-in `ViewController` with the custom controller in a specific situation.
-- **Replace** the `ViewController` with the custom controller for the whole bundle.
+- Configure a custom controller alongside regular matcher rules to use **both** your custom controller and the `ViewController`.
+- [Add a listener to add your custom logic](#adding-a-listener) before or after the view is rendered.
+- [**Override**](#overriding-the-built-in-viewcontroller) the built-in `ViewController` with the custom controller in a specific situation (not recommended).
```

Comment:
This is "custom Symfony controller" right?

Reply:
I'm actually wondering if I'd mention it at all. Yes, it can be done, but why ? If you want to do something for every content view, the event listener can be used for that. I'd rather not mention it at all if it doesn't add any value.

Overriding services is something you can always do, it is not limited to this part of the system.

Reply:
Actually, this part isn't about doing something for every content view. The part you most probably have in mind is called "Overriding the built-in ViewController" and I removed it because I had the same thoughts as you.
This part describes applying the custom controller in a given match situation by indicating the controller, but no template. 
@bdunogier if I understand correctly, you have described this as "The way to customize what a content view does". However, as there is no template provided, you either need to provide one inside the controller or return a {{Response}} instead of the {{ContentView}}. I'd argue if it wouldn't be a better idea to use the first way by actually providing the template inside the configuration (if it is known). This way it's easy to just return {{ContentView}} object and let the Listener do the rendering (and this is the first way after my changes).
I'd probably also argue that if I'd like to modify the rendered {{Response}} object, then I'd also let the Listener do the rendering and then modify the {{Response}} in my own Listener, as I have described in the "Adding a listener" part (and this is the second way after my changes). I think that would be preferable to doing the rendering myself in the controller. 
However, I'm not so experienced with this topic, so I would like your opinion on the matter :)

Reply:
> I'd probably also argue that if I'd like to modify the rendered Response object, then I'd also let the Listener do the rendering and then modify the Response in my own Listener

I'd actually pre-configure the `Response` using `setResponse()`.

Reply:
@bdunogier You are right, I haven't thought about this option. I added an example of how to do this. Thanks.